### ISSUE TITLE
Fix grammatically incorrect feedback in the "What Is the Purpose of WAI-ARIA" lecture

### ISSUE THREAD

**Issue by majestic-owl448** [freeCodeCamp/freeCodeCamp#68234]
### Describe the Issue

1. In the first question's wrong-answer feedback (which appears three times, identically), the phrase "HTML elements accessibility" is grammatically incorrect (a noun pile-up missing a possessive or preposition):

> Focus on how WAI-ARIA enhances HTML elements accessibility.

### Affected Pages

https://www.freecodecamp.org/learn/responsive-web-design-v9/lecture-introduction-to-aria/what-is-the-purpose-of-wai-aria

### Expected behavior

1. Reword the feedback (in all three places) to `Focus on how WAI-ARIA enhances the accessibility of HTML elements.`

**Comment by majestic-owl448** [comment]
This looks like something that can be fixed by "first-time" code contributors to this repository. These are the lines you should be looking at to work on a fix:

https://github.com/freeCodeCamp/freeCodeCamp/blob/71b316cc10791a9d1bb19f02d5cfd5141f762c7a/curriculum/challenges/english/blocks/lecture-introduction-to-aria/672a53cf67140d8cd85d4b0f.md#L111
https://github.com/freeCodeCamp/freeCodeCamp/blob/71b316cc10791a9d1bb19f02d5cfd5141f762c7a/curriculum/challenges/english/blocks/lecture-introduction-to-aria/672a53cf67140d8cd85d4b0f.md#L123
https://github.com/freeCodeCamp/freeCodeCamp/blob/71b316cc10791a9d1bb19f02d5cfd5141f762c7a/curriculum/challenges/english/blocks/lecture-introduction-to-aria/672a53cf67140d8cd85d4b0f.md#L131

Please make sure you read our [guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in our [chat room](https://discord.gg/PRyKn3Vbay) or our [forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing; our moderators will guide you through this.

Sometimes we may get more than one pull request. We typically accept the most quality contribution followed by the one that is made first.

Happy contributing.
